embark.el

```diff
@@ -949,7 +949,11 @@ their own target finder.  See for example
 
 (defun embark--cycle-key ()
   "Return the key to use for `embark-cycle'."
-  (or embark-cycle-key (car (where-is-internal #'embark-act))))
+  (if embark-cycle-key
+      (if (key-valid-p embark-cycle-key)
+          (key-parse embark-cycle-key)
+        (error "`embark-cycle-key' is invalid"))
+    (car (where-is-internal #'embark-act))))
```

Comment:
I think this is wrong: you later call `keymap-set` with this key and `where-is-internal` does not return strings that are `key-valid-p`, which is what `keymap-set` takes.